semantic-core/src/Analysis/ImportGraph.hs

```diff
@@ -68,39 +67,44 @@ importGraph eval
   . traverse (runFile eval)
 
 runFile
-  :: ( Carrier sig m
+  :: forall term name m sig
+  .  ( Carrier sig m
```

Comment:
Most (all?) of the `runFile` functions need `ScopedTypeVariables` now since they have to converge terms, initialize heaps abstracted over the name type, etc.